doc/radosgw/multisite.rst

```diff
@@ -105,28 +105,24 @@ have the following pools:
 
 -  ``us-east.rgw.control``
 
--  ``us-east.rgw.data.root``
-
--  ``us-east.rgw.gc``
+-  ``us-east.rgw.meta``
 
 -  ``us-east.rgw.log``
 
--  ``us-east.rgw.intent-log``
-
--  ``us-east.rgw.usage``
-
--  ``us-east.rgw.users.keys``
-
--  ``us-east.rgw.users.email``
-
--  ``us-east.rgw.users.swift``
-
--  ``us-east.rgw.users.uid``
-
 -  ``us-east.rgw.buckets.index``
 
 -  ``us-east.rgw.buckets.data``
 
+The zone definitions (see ``radosgw-admin zone get``) list several more
```

Comment:
yes, probably the pools section in its own page would be the best we could just say <zone>.pool names and list them, and in case of single site deployments, these would be called default as that is the default zone name?